docs/components/gantry/single-axis.md

```diff
@@ -61,9 +57,8 @@ The following attributes are available for `single-axis` gantries:
 | --------- | ---- | --------- | ----------  |
 | `length_mm` | int | **Required** | Length of the axis of the gantry in millimeters. |
 | `motor` | string | **Required** | `name` of the [motor](/components/motor/) that moves the gantry's actuator. |
-| `axis` | object | **Required** | The translational axis for the gantry. Must be exactly one of `x`, `y`, or `z`. |
 | `board`  |  Optional | string | `name` of the [board](/components/board/) containing the limit switches and pins. If `limit_pins` exist, `board` is required. |
 | `limit_pins`  | Optional | object | The `boards`'s pins attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), `limit_pins` are required to be set. |
 | `limit_pin_enabled_high` | boolean | Optional | Whether the limit pins are enabled. <br> Default: `false` |
-| `mm_per_rev` | int | Optional | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
-| `gantry_rpm` | int | Optional | The gantry `motor`'s default revolutions per minute (RPM). |
+| `mm_per_rev` | int | **Required** | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
+| `gantry_mm_per_sec` | int | Optional | The speed at which the gantry moves in millimeters per second. Used to calculate the gantry `motor`'s revolutions per minute (RPM). It defaults to 100 RPM. |
```

Comment:
```suggestion
| `gantry_mm_per_sec` | int | Optional | The speed at which the gantry moves in millimeters per second. Used to calculate the gantry `motor`'s revolutions per minute (RPM). <br> Default: `100` |
```
for docs formatting-- unless (q) I'm confused-- is the value itself a derivative of some kind to get 100 rpm?

Reply:
yeah whatever value the user puts in we convert to RPM because that's what the motor needs, so the default basically whatever equals 100 rpm, but that will be different based on each users setup

Reply:
added RPM to your suggestion... does that work @sguequierre?